src/components/organisms/VideoModal/VideoModal.tsx

```diff
@@ -22,34 +24,32 @@ export const VideoModal: React.FunctionComponent<PropsType> = ({
   url,
   caption,
   autoplay = false,
-  backdrop = "static",
 }) => {
   const closeVideoModal = () => {
     onHide();
   };
 
   return (
-    <Modal
-      show={show}
-      onHide={closeVideoModal}
-      centered={true}
-      size={"xl"}
-      backdrop={backdrop}
-    >
-      <Modal.Header closeButton>
-        {caption && <Modal.Title>{caption}</Modal.Title>}
-      </Modal.Header>
-      <div className="video-modal-container">
-        <div className="content">
+    <Modal isOpen={show} onClose={closeVideoModal} fullWidth>
+      <div className="VideoModal">
+        {caption && <h3>{caption}</h3>}
```

Comment:
components like `Modal.Title` should not be replaced with `h3` or `h` with any number, but a `<div>` with maybe a class like `<div className="VideoModal__title">`

Reply:
It will also be not out of place to have a custom `<ModalTitle>` component that uniforms all modal titles into the same style. Removing Bootstrap doesn't necessarily mean all patterns used by it are bad.

Reply:
didn't create a `title` component as all titles are different, but added some custom styles + className to this now `div` element c90aa28